src/styles/watch.scss

```
@@ -0,0 +1,63 @@
+
+.watch {
+  position: relative;
```

Position for bem block? 🥲

```suggestion
.watch {
  position: relative;
```

---

Додала обгортку і перенесла Position туди.